native-plasma/app/src/main/cpp/plasma.c

```diff
@@ -496,6 +496,8 @@ static void engine_handle_cmd(struct android_app* app, int32_t cmd) {
 }
 
 void android_main(struct android_app* state) {
+  app_dummy();
```

Comment:
app_placeholder

Reply:
Well, app_placeholder is not a thing.  However, upon further reading I realized that app_dummy is deprecated and the use of the linker export flag is the recommended way.  So I reverted that part of the change.